bigquery/indexer.py

```diff
@@ -123,13 +123,11 @@ def main():
   es = init_elasticsearch(args.elasticsearch_url, index_name)
 
   f = open(os.path.join(args.config_dir, 'facet_fields.csv'))
-  rows = csv.DictReader(filter(lambda row: row[0]!='#', f),
-      # field_name is BigQuery field name.
-      # readable_field_name is used in index and Data Explorer UI.
-      fieldnames=['project_id', 'dataset_id', 'table_name', 'field_name', 'readable_field_name'],
-      skipinitialspace=True)
+  # Remove comments using jsmin.
+  csvString = jsmin.jsmin(f.read())
```

Comment:
Looks like this is all done in-memory, wondering if this might become a problem with very large CSVs. What's the upper bound on file size that we can reasonably expect to see?

Reply:
This file stores facets. There will be at most dozens of facets.